ark/phenotyping/som_utils.py

```diff
@@ -468,19 +587,26 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
         )
 
         # load segmentation labels in for fov
+        # CANDACE SUGGESTION: perhaps set '_feature_0' as some global constant
```

Comment:
Agreed, can set this as a global, and default to the global for an arg in  create_pixel_matrix